src/sagemaker/estimator.py

```diff
@@ -349,6 +349,9 @@ def __init__(
         self.profiler_config = profiler_config
         self.disable_profiler = disable_profiler
 
+        if not _region_supports_profiler(self.sagemaker_session.boto_region_name):
```

Comment:
Do you think a test exercising this branch is warranted?

Reply:
Have been passed in Unit and integ test (mxnet using default profiler confit in test profiler) from my side. Do you think it's necessary to test the branch in environment of China regions?